docs(edge-services): typo and minor fixes MTA-7306 (#6683)
scaleway/docs-content: pages/edge-services/concepts.mdx

Refer to [CNAME records for Edge Services](/edge-services/reference-content/cname-record/) for more information.

## Destination host

> In the case of a Load Balancer as an Edge Services backend, the specific host for which Edge Services requests and caches data. This is an optional setting: when specified, this host (e.g. `mydomain.com`) is used in the HTTP `Host` header when Edge Services requests data from the Load Balancer. If no destination host is specified, the `Host` from the incoming request will be used.

Reviewer: ```suggestion
In the case of a Load Balancer as an Edge Services backend, the specific host for which Edge Services requests and caches data. This is an optional setting: when specified, this host (e.g. `mydomain.com`) is used in the HTTP `Host` header when Edge Services requests data from the Load Balancer. If no destination host is specified, the `Host` from the incoming request is used.
```

> The destination host must be associated with the Load Balancer / its backend servers, and only one host may be set per pipeline. If your Load Balancer is in front of multiple hosts, you can create a separate Edge Services pipeline for each. Each host will therefore get its own Edge Services endpoint and cache.

Reviewer: ```suggestion
The destination host must be associated with the Load Balancer / its backend servers. You can set only one host per pipeline. If your Load Balancer is in front of multiple hosts, you can create a separate Edge Services pipeline for each. Each host therefore gets its own Edge Services endpoint and cache.
```

Reviewer: Not sure about the first sentence, I don't get what the slash means here. Maybe you can rewrite without the slash.

> The destination host must be associated with the Load Balancer / its backend servers, and only one host may be set per pipeline. If your Load Balancer is in front of multiple hosts, you can create a separate Edge Services pipeline for each. Each host will therefore get its own Edge Services endpoint and cache.

Author: ```suggestion
The destination host must be associated with the Load Balancer (or its backend servers). You can set only one host per pipeline. If your Load Balancer is in front of multiple hosts, you can create a separate Edge Services pipeline for each. Each host therefore gets its own Edge Services endpoint and cache.
```

The endpoint from which a given Edge Services pipeline can be accessed, e.g. `https://pipeline-id.svc.edge.scw.cloud`. When a client requests content from the Edge Services endpoint, it is served by Edge Services and its cache, rather than from the backend (Object Storage bucket or Load Balancer) directly. Edge Services automatically manages redirection from HTTP to HTTPS.

> The endpoint can be customized with a user-defined subdomain, allowing you to replace the standardized endpoint with the subdomain of a domain you already own, e.g. `http://my-own-domain.com`. Using customized endpoints requires an associated [certificate](#certificate), and [CNAME record](#cname-record).

Reviewer: ```suggestion
You can customize the endpoint with a user-defined subdomain, replacing the standardized endpoint with the subdomain of a domain you already own — e.g., `http://my-own-domain.com`. Using customized endpoints requires an associated [certificate](#certificate), and [CNAME record](#cname-record).
```

An Edge Services pipeline consists of a [backend](#backend), which Edge Services can protect from threats with a [Web Application Firewall](#waf), and for which it also requests and [caches](#cache) content. Each pipeline also has an [endpoint](#endpoint) from which content is accessed and served via Edge Services. The pipeline's endpoint can be customized with a user-defined [subdomain](/domains-and-dns/concepts/#subdomain) and associated [certificate](#certificate) so that Edge Services can serve content over HTTPS.

> You can create an Edge Services pipeline for each of your Object Storage buckets or Load Balancers. Note that caching, WAF, and other optional parts of the pipeline can be enabled and disabled at will. The customization of the endpoint is also optional.

Reviewer: ```suggestion
You can create an Edge Services pipeline for each of your Object Storage buckets or Load Balancers. You can enable and disable caching, WAF, and other optional parts of the pipeline at any time. Customizing the endpoint is also optional.
```
